maint-lib/stagelib/envglobals.py

```diff
@@ -15,6 +15,7 @@
 # Allow lines to extend beyond 99 char limit by 6 chars to support output formatting to 100 cols
 REQUIRED_ENV_VARS = OrderedDict([
     ('CEPH_VERSION',      'Ceph named version being built (e.g., luminous, mimic)'),
+    ('CEPH_POINT_RELEASE', 'Ceph point release version (e.g., 12.2.0)'),
```

Comment:
use a dash instead of the comma `e.g: -12.2.0`, and highlight the fact we need a dash depending on the distro.